main.go

```
@@ -51,9 +51,17 @@ func init() {
 	// +kubebuilder:scaffold:scheme
 }
 
+func getWorkloadImage() string {
+	if workload, exist := os.LookupEnv("WORKLOAD_IMAGE"); exist {
+		return workload
+	}
+	return "quay.io/freeipa/freeipa-openshift-container:freeipa-server"
```

It would be better to raise an error if no value is provided.